Templates/CRMs/Magento/Magento Order to Salesforce/long-description.md

```diff
@@ -0,0 +1,23 @@
+This template exports sales orders from Magento and imports them to Salesforce.
+Using this template, you can insert sales order and order products data from Magento to Salesforce.
+
+![Template](assets/Adobe_Commerce___Magento_Order_to_Salesforce.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Salesforce user should have access to client ID, secret and security token.
```

Comment:
User does not have to have access, you just need that info for the template to work. So just "Salesforce user exists for performing the synchronization" and "Salesforce app exists for performing the synchronization".

Reply:
Fixed!